rockspec/apisix-master-0.rockspec

```diff
@@ -43,7 +43,7 @@ dependencies = {
     "lua-resty-cookie = 0.1.0",
     "lua-resty-session = 2.24",
     "opentracing-openresty = 0.1",
-    "lua-resty-radixtree = 2.6",
+    "lua-resty-radixtree = 2.6.1",
```

Comment:
`lua-resty-expr` also released v1.1.0, I think it can be updated by the way.

Reply:
Better to use separate PR to update lua-resty-expr, as the fix doesn't require to change it.

Reply:
Yes.